docs/releases/v0.23.0.rst

```diff
@@ -19,13 +19,23 @@ Maintenance
 Upgrade considerations
 ----------------------
 
+* Wagtail dropped support for Python 3.6. Upgrade your Python to 3.7 and above.
+
+* Wagtail dropped support for Django 3.0, 3.1, and 3.2. Upgrade your Django to 3.2 and above.
+
+* Wagtail is now using a new search backend.
+
+    * Specify the `new default in your settings <https://docs.wagtail.org/en/stable/releases/2.15.html#database-search-backends-replaced>`_.
+    * Next, you will need to run the `update_index command <https://docs.wagtail.org/en/stable/reference/management_commands.html#update-index>`_.
+
+* Other upgrade considerations for `Wagtail 2.15 <https://docs.wagtail.org/en/stable/releases/2.15.html#upgrade-considerations>`_ and `Wagtail 2.16 <https://docs.wagtail.org/en/stable/releases/2.16.html#upgrade-considerations>`_ may apply to your site.
```

Comment:
Did you run into any migration issues, i.e. does `makemigrations` product any output or try to make anything?

Reply:
No migrations for codered cms. All the migrations ran in my local wagtail package.